infrastructure-playbooks/purge-cluster.yml

```
@@ -42,12 +42,130 @@
     - "{{ nfs_group_name|default('nfss') }}"
     - "{{ client_group_name|default('clients') }}"
     - "{{ mgr_group_name|default('mgrs') }}"
+    - ceph-grafana
```

```suggestion
    - grafana
```

to keep consistency with other role names

---

I suppose it should be `grafana-server` then.